hack/rpm-deps.sh

```diff
@@ -7,7 +7,7 @@ source hack/bootstrap.sh
 source hack/config.sh
 
 LIBVIRT_VERSION=${LIBVIRT_VERSION:-0:9.5.0-6.el9}
-QEMU_VERSION=${QEMU_VERSION:-17:8.0.0-13.el9}
+QEMU_VERSION=${QEMU_VERSION:-17:8.2.0-2.el9}
```

Comment:
@andreabolognani is it fine to upgrade to this QEMU version? Usually, we do all the virtualization components together

Reply:
I can also update the other components, that was just the one RPM that was already missing from the mirror.

Reply:
I would do this in a separate PR

Reply:
While i don't mind putting this in a separate PR, i would like to only have to do it once, as the other arches already have the RPMs cached on google storage.
I would therefore propose to leave it as is until this PR is approved and then raise the PR to bump the QEMU version.

Reply:
I was waiting for libvirt 10.0.0 to be available in CentOS Stream before doing the usual round of updates.

The build has been done, but based on past experience that could take a couple of weeks or even more to actually show up in mirrors. I think it would be unfair to Jan to hold up this PR for that long.

What's the schedule for the next KubeVirt release? I would prefer not to have a half-updated situation in a tagged release, but as a temporary state it could be fine as long as all the functional tests pass.

Another alternative could be to temporarily define `$QEMU_VERSION_S390X` and leave other architectures alone.

Reply:
@jschintag do you need this PR to be merged before doing the s390x support? I'm trying to understand if we can indeed wait for libvirt 10.0.0

Reply:
Well we do need this PR for s390x support, but i don't think waiting with merging this would mean we can't do other work for s390x support (mainly testing this further and enabling other repos like CDI).
That being said, would it also be an option to update to `qemu-img-8.1.0-4` and libvirt `9.9.0-1` instead? I don't really need a specific version after all, i just need it to be available from the mirror.

Reply:
You can enable CDI independently from `kubevirt/kubevirt`.

If possible I would avoid to have mismatches between architectures and a different version for QEMU for s390x. Otherwise, we can introduce a temporary env variable as @andreabolognani suggested if this is taking too long.

Once we have the support merged then the s390x rpms will be cached and we shouldn't encounter this problem anymore.

Reply:
> That being said, would it also be an option to update to `qemu-img-8.1.0-4` and libvirt `9.9.0-1` instead? I don't really need a specific version after all, i just need it to be available from the mirror.

No, it's fine to pick the latest QEMU build and leave libvirt alone.

There's really no nice way to handle this, so we have to pick among bad options :) The good news is that at least it's going to be a temporary situation, so in the not-that-long run it won't matter.

Anyway, I see that there is some other discussion going on in the PR, so this is not the only thing blocking a merge. Once we get to the point where that's the case, we will have to make a call.

Reply:
Agree. I think my preference in that case would be to have a separate Variable for s390x that starts with `TMP_` and has big comments on it to drop it when doing the next upgrade.

Reply:
I have put the s390x rpm versions into separate variables for now, so that they do not interfere with the other arches.